src/data_provider/src/network/networkLinuxWrapper.h

```diff
@@ -190,10 +190,10 @@ class NetworkLinuxInterface final : public INetworkInterfaceWrapper
         }
         return retVal.get();
     }
- 
+
     static std::string getRedHatDHCPStatus(const std::vector<std::string>& fields)
     {
-        std::string retVal { UNKNOWN_VALUE };
+        std::string retVal { "enabled" };
```

Comment:
is correct this default value ?

Reply:
Yes! This function is being called in the `dhcp()` one which initializes the retVal in "unknown"